iOSCoreSample/iOSCoreSample/AppDelegate.swift

```diff
@@ -0,0 +1,23 @@
+import PushKit
+import RealtimeKit
+import UIKit
+
+@main
+class AppDelegate: UIResponder, UIApplicationDelegate, PKPushRegistryDelegate {
```

Comment:
```suggestion
class AppDelegate: UIResponder, UIApplicationDelegate {
```

`PKPushRegistryDelegate` is related to the CallManager sample, VOIP notification. We should remove the `pushRegistry` code.